bin/standalone-installer-unix

```diff
@@ -0,0 +1,175 @@
+# desired version as the first argument to this program.
+#
+set -euo pipefail
+shopt -s failglob
+
+: "${NEXTSTRAIN_DOT_ORG:=https://nextstrain.org}"
+
+# Globals declared here to make them more obvious, but set by main() to avoid
+# source ordering requirements and delay execution until the full file is
+# parsed.
+declare KERNEL TARGET DESTINATION VERSION
+
+# Wrap everything in a function which we call at the end to avoid execution of
+# a partially-downloaded program.
+main() {
+    KERNEL="${KERNEL:-$(uname -s)}"
+    TARGET="${TARGET:-$(target-triple)}"
+    DESTINATION="${DESTINATION:-$(destination)}"
+
+    VERSION="${1:-${VERSION:-latest}}"
+
+    local archive archive_url tmp
+
+    archive="standalone-${TARGET}.tar.gz"
+    archive_url="${NEXTSTRAIN_DOT_ORG}/cli/download/${VERSION}/${archive}"
+
+    # Move into a temporary working dir
+    tmp="$(mktemp -d)"
+    if ! debug; then
+        trap "rm -rf $tmp" EXIT
+    fi
+    pushd "$tmp" >/dev/null
+    log "Temporary working directory: $tmp"
+
+    log "Downloading $archive_url"
+    curl "$archive_url" \
+        --fail --show-error --location --proto "=https" \
+        > "$archive"
+
+    log "Extracting $archive"
+    mkdir standalone
+    tar xz$(if-debug v)f "$archive" -C standalone
+
+    if [[ -d "$DESTINATION" ]]; then
+        log "Removing existing $DESTINATION"
+        rm -rf "$DESTINATION"
+    fi
+
+    log "Installing to $DESTINATION"
+    mkdir -p "$(dirname "$DESTINATION")"
+    mv standalone "$DESTINATION"
+
+    popd >/dev/null
+
+    if ! debug; then
+        log "Cleaning up"
+        rm -rf "$tmp"
+    fi
+
+    cat <<~~
+______________________________________________________________________________
+
+Nextstrain CLI ($("$DESTINATION"/nextstrain --version)) installed to $DESTINATION.
+
+~~
+
+    if [[ "$(type -p nextstrain 2>/dev/null)" != "$DESTINATION/nextstrain" ]]; then
+        cat <<~~
+To make the "nextstrain" command available without using its full path,
+please add the following to your shell initialization file (e.g. ~/.bashrc or
+~/.zshrc):
+
+    export PATH="$DESTINATION:\$PATH"
+
+and then logout and login again.  You only need to do this once.
+
+Your default shell appears to be $(default-shell).
+~~
+    fi
+}
+
+target-triple() {
+    local machine vendor os
+
+    machine="$(uname -m)"
+
+    case "$KERNEL" in
+        Linux)
+            vendor=unknown
+            os=linux-gnu
+            ;;
+
+        Darwin)
+            vendor=apple
+            os=darwin
+            ;;
+
+        *)
+            die "unknown kernel: $KERNEL"
+    esac
+
+    echo "$machine-$vendor-$os"
+}
+
+destination() {
+    local user_data dest
+
+    # Paths from appdirs <https://github.com/ActiveState/appdirs/blob/master/appdirs.py>
+    case "$KERNEL" in
+        Linux)
+            user_data="${XDG_DATA_HOME:-$HOME/.local/share}";;
+
+        Darwin)
+            user_data="$HOME/Library/Application Support";;
```

Comment:
```suggestion
            user_data="$HOME/Library/Application\ Support";;
```

Need to escape whitespace, otherwise this won't work with copy-paste

Reply:
I don't follow. Copying what from where?

Reply:
Sorry my bad. My suggestion doesn't actually work.

The problem is that the blurb on how to add the command to PATH didn't work when I copied without `"`.

But that was my mistake. Both of the following work:
```
/Users/corneliusromer/Library/Application\ Support/nextstrain/cli/standalone/nextstrain   
"/Users/corneliusromer/Library/Application Support/nextstrain/cli/standalone/nextstrain"
```

But the following do not work:
```
/Users/corneliusromer/Library/Application Support/nextstrain/cli/standalone/nextstrain   
"/Users/corneliusromer/Library/Application\ Support/nextstrain/cli/standalone/nextstrain"
```

Reply:
Talking about this part <img width="1645" alt="image" src="https://user-images.githubusercontent.com/25161793/187305038-4a7f9839-ddbb-43af-8e94-0b977326faa5.png">

Reply:
Ah, got it. No worries.